src/components/Header.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import { Navigation } from 'hds-react';
+
+import { useClient } from '../clients/client';
+
+type LanguageOption = { code: string; label: string };
+
+const Header = (): React.ReactElement => {
+  const languageOptions = [
+    { code: 'fi', label: 'Suomi' },
+    { code: 'sv', label: 'Svenska' },
+    { code: 'en', label: 'English' }
+  ];
+
+  const client = useClient();
+  const authenticated = client.isAuthenticated();
+  const initialized = client.isInitialized();
+  const user = client.getUser();
+
+  const [language, setLanguage] = useState(languageOptions[0]);
+  const [active, setActive] = useState<'link' | 'button' | 'dropdown'>();
+
+  // show helsingfors logo if swedish is selected as the language
+  const logoLanguage = language.code === 'sv' ? 'sv' : 'fi';
+
+  const title = 'Helsinki Profiili Example';
+
+  // formats the selected value
+  const formatSelectedValue = ({ code }: LanguageOption): string =>
+    code.toUpperCase();
```

Comment:
There are a few occurrences in this file where a comment repeats exactly the next line, without giving any additional information. IMO that's useless, I would remove such comments.

Reply:
I didnt like 'em either. :) Came with the file from profile-ui. Removed.